scripts/benchmarks_monthly.py

```diff
@@ -75,7 +75,13 @@ def add_arguments(parser: ArgumentParser) -> ArgumentParser:
         '--no-clean',
         dest='no_clean',
         action='store_true',
-        help='Do not clean the SDK and results directories before execution')
+        help='Do not clean the SDK installations before execution')
+
+    parser.add_argument(
+        '--resume',
+        dest='resume',
+        action='store_true',
+        help='Resume a previous run from existing benchmark results')
```

Comment:
attempt to .. ?

Reply:
It _will_ resume from the previous results--running any benchmarks where there aren't existing, valid results. It's possible it will re-run some, but only when their results were incomplete/corrupt/invalid.